.github/workflows/build.yml

```diff
@@ -88,8 +96,8 @@ jobs:
         if: runner.os != 'Windows'
         uses: urkle/action-get-ninja@v1
 
-      - name: Prepare AppImage (Linux)
```

Comment:
Keep the `(Linux)` pls. It looks nice in the workflow views.

Reply:
opz, that was a mistake, fixing now